### ISSUE TITLE
Exception in Apex Logs Cloud Run service

### ISSUE THREAD

**Issue by steren** [apex/roadmap#48]
I see an recurring exception in the Cloud Run service used by Apex Logs:

```
2020/08/05 04: 48:01 http: panic serving 169.254.8.129:42092: runtime error: invalid memory address or nil pointer dereference
at net/http.(*conn).serve.func1 (server.go:1772)
at panic (/usr/local/go/src/runtime/panic.go:975)
at github.com/tj/go-buffer.(*Buffer).FlushSync (buffer.go:152)
at github.com/apex/log/handlers/apexlogs.(*Handler).FlushSync (apexlogs.go:107)
at github.com/tj/logs/internal/usage.Flush (usage.go:63)
at main.alerting.func1 (main.go:155)
at net/http.HandlerFunc.ServeHTTP (server.go:2012)
at main.reporting.func1 (main.go:189)
at net/http.HandlerFunc.ServeHTTP (server.go:2012)
at net/http.serverHandler.ServeHTTP (server.go:2807)
at net/http.(*conn).serve (server.go:1895)
```

**Comment by tj** [comment]
That's embarrassing hahah, will get this fixed right away! Thanks

**Comment by tj** [comment]
Fixed! If anyone hits this, update to v0.9.7 with:

```
$ logs upgrade && logs instance update
```
